opslevel/resource_opslevel_integration_azure_resources.go

```diff
@@ -116,17 +115,24 @@ func (r *IntegrationAzureResourcesResource) Schema(ctx context.Context, req reso
 				Description: "The name of the integration.",
 				Required:    true,
 			},
-			"ownership_tag_keys": schema.SetAttribute{
+			"ownership_tag_keys": schema.ListAttribute{
 				ElementType: types.StringType,
-				Description: "An Array of tag keys used to associate ownership from an integration. Max 5",
+				Description: "An Array of tag keys used to associate ownership from an integration. Max 5 (default = [\"owner\"])",
 				Optional:    true,
-				Validators: []validator.Set{
-					setvalidator.SizeBetween(1, 5),
+				Computed:    true,
+				// current API default below
+				Default: listdefault.StaticValue(types.ListValueMust(types.StringType, []attr.Value{types.StringValue("owner")})),
```

Comment:
If the user didn't have this field set but via the UI they used an array of `["owner", "service", "app"]` I think this would reset it back to `["owner"]`

Reply:
Yes, this is correct. This also correctly shows up during `terraform plan` and setting these tags is non-trivial so I'm not sure if we need to do anything extra here 🤔